### PR TITLE
Updated tests

### PR DESCRIPTION
**Pull request by terakilobyte**
@SaintPeter could you see if this reporter setup works on windows?

I've been looking into tape a bit more deeply and am favoring this reporter and style over others. It's a hybrid of Eric Elliot's suggested style and that of the tape documentation itself.

What do you all think?

### CODE REVIEW

File: tests/test.spec.js [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
+    const actual = typeof Array.prototype.map;
+    const expected = 'object';
 
-    // t.pass(); This would generate a passing assertion
-    t.comment('Assertion passes');
-    t.equals(actual, expected);
-    // t.fail(); This would generate a failing assertion
-    t.comment('Assertion fails');
-    t.notEqual(actual, expected);
+    t.equals(actual, expected, 'typeof Array.prototype.map is "object"');
```
Comment (BerkeleyTrue) [inline tests/test.spec.js]: We don't need to test this. Why is this here?

File: src/Components/Editor.js [hunk 2]
```diff
@@ -93,18 +93,18 @@ class Editor extends Component {
...
         case 'solutions':
-          if (challengeType !== 5 && challengeType !== 1) {
-            mode = 'htmlmixed';
-            break;
-          }
+        if (challengeType !== 5 && challengeType !== 1) {
+          mode = 'htmlmixed';
+          break;
+        }
         case 'head':
```
Comment (BerkeleyTrue) [inline src/Components/Editor.js:100]: The indenting on this switch statement should be preserved.

File: tests/test.spec.js (hunk 1 above)
Comment (terakilobyte) [inline tests/test.spec.js]: It's more an example of testing in general than something that needs to be there.

**Comment by arialaurel** [comment]
Tests work for me on Windows. @SaintPeter does this get them working for you?

**Comment by SaintPeter** [comment]
Looks good:

```
$ npm test

> COM1000@0.0.0 test d:\mean\COM1000
> babel-tape-runner tests/**/*.js | colortape

TAP version 13
# reducer.spec
# reducer should return a default state if none is provided
ok 1 reducer returns a default state
# reducer should return a new state upon a valid action type
ok 2 reducer returns a new state if action is valid
# reducer should return previous state on invalid action
ok 3 reducer returns previous state if the action is invalid

1..3
# tests 3
# pass  3

# ok
```

**Comment by BerkeleyTrue** [comment]
:+1:
